pom.xml

```diff
@@ -2007,6 +2009,22 @@
                 <artifactId>javassist</artifactId>
                 <version>3.29.2-GA</version>
             </dependency>
+            <dependency>
+                <groupId>org.junit.platform</groupId>
+                <artifactId>junit-platform-launcher</artifactId>
```

Comment:
Why do we need this dependency?

Reply:
Correct we don't need platform launcher dependency. Removed it : f711670b6d7fd685f3192a0041c19ca5ac9473df